.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "all",
```

Comment:
Probably "es5" is just good enough? I find it weird seeing trailing commas in the last parameter of functions. 

This may require us to update the VS Code settings file to match the rule.

WDYT?

Reply:
Yep, I prefer this as well. Google style guidelines recommend "all", but the only relevant thing is, that the options we choose match throughout the source code...